invenio/base/scripts/demosite.py

```diff
@@ -94,10 +96,9 @@ def populate(packages=[], default_data=True, files=None,
     from werkzeug.utils import import_string
     map(import_string, packages)
 
-    from invenio.ext.sqlalchemy import db
     print(">>> Going to load demo records...")
-    db.session.execute("TRUNCATE schTASK")
```

Comment:
the only way I know is to use the [truncate_table](https://github.com/inveniosoftware/invenio/blob/9d71513860f25751d8378c5e51707609db4c0600/invenio/legacy/dbquery_postgresql.py#L432) function.